pkg/render/node.go

```diff
@@ -246,6 +246,12 @@ func (c *nodeComponent) nodeRole() *rbacv1.ClusterRole {
 		},
 
 		Rules: []rbacv1.PolicyRule{
+			{
+				// Calico uses endpoint slices for service-based network policy rules.
+				APIGroups: []string{"discovery.k8s.io"},
+				Resources: []string{"endpointslices"},
+				Verbs:     []string{"get", "list", "watch"},
```

Comment:
Do we need `get` ?  Your docs PR only has `list` and `watch`, and I only appeared to need `list` and `watch` in my similar change.

Reply:
oh yeah, hm. Probably not. I think this was a copy/paste from elsewhere.